js/main.js

```diff
@@ -0,0 +1,133 @@
+const coll = document.querySelectorAll('.collapsible');
+const $menu = document.querySelector('.menu');
+const $myLinks = document.querySelector('.my-links');
+const $set = document.querySelector('.expansion-sets');
+const $viewHome = document.querySelector('.view-home');
+const $viewSet = document.querySelector('.view-set');
+const $tab = document.querySelectorAll('.tab');
+const $deckColumn = document.querySelector('.deck-column');
+const $header = document.querySelector('header');
+const $setLogo = document.querySelector('.set-logo');
+const $cardWrapper = document.querySelector('.card-wrapper');
+const $setList = document.querySelectorAll('.set-list');
+
+const cardSets = new XMLHttpRequest();
+cardSets.open('GET', 'https://api.pokemontcg.io/v2/sets');
+cardSets.responseType = 'json';
+cardSets.addEventListener('load', function () {
+  for (let i = 0; i < cardSets.response.data.length; i++) {
+    if (cardSets.response.data[i].series === 'Scarlet & Violet') {
+      const $setWrapper = createSet(i);
+      $setList[0].appendChild($setWrapper);
+    }
+    if (cardSets.response.data[i].series === 'Sword & Shield') {
+      const $setWrapper = createSet(i);
+      $setList[1].appendChild($setWrapper);
+    }
+    if (cardSets.response.data[i].series === 'Sun & Moon') {
+      const $setWrapper = createSet(i);
+      $setList[2].appendChild($setWrapper);
+    }
```

Comment:
I would turn the last 2 `if`s into `else if`s. By having these all as `if` statements, you're forcing us to check all 3 conditionals every time, even if only the first one is true. By turning the last 2 into `else if`s, as soon as we find the correct conditional, we won't check the rest